test/CodeToRest_test.py

```diff
@@ -636,11 +649,580 @@ def test_35(self):
+                '# Comment here\n',
+                bf +
+                ' DetailPrint "Hello World!"\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['NSIS'])
+
+    # TeX.
+    def test_72(self):
+        self.mt('Hello World!\n'
+                '% Comment here\n',
+                bf +
+                ' Hello World!\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['TeX'])
+
+    # Erlang.
+    def test_73(self):
+        self.mt('Hello World!.\n'
+                '% Comment here\n',
+                bf +
+                ' Hello World!.\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Erlang'])
+
+    # QBasic.
+    def test_74(self):
+        self.mt('PRINT "Hello World!";\n'
+                '\' Comment here\n',
+                bf +
+                ' PRINT "Hello World!";\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['QBasic'])
+
+    # VB.net.
+    def test_75(self):
+        self.mt('Console.WriteLine("Hello World!")\n'
+                '\' Comment here\n',
+                bf +
+                ' Console.WriteLine("Hello World!")\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['VB.net'])
+
+    # REBOL.
+    def test_76(self):
+        self.mt('view [text "Hello World!"]\n'
+                '; Comment here\n',
+                bf +
+                ' view [text "Hello World!"]\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['REBOL'])
+
+    # LLVM.
+    def test_77_a(self):
+        self.mt('; Comment here\n',
+                sl(-3) +
+                'Comment here\n', ['LLVM'])
+
+    def test_77_b(self):
+        self.mt('Hello World!\n'
+                '; Comment here\n',
+                bf +
+                ' Hello World!\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['LLVM'])
+
+    # Ada.
+    def test_78(self):
+        self.mt('Put_Line("Hello World!");\n'
+                '-- Comment here\n',
+                bf +
+                ' Put_Line("Hello World!");\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Ada'])
+
+    # Eiffel.
+    def test_79(self):
+        self.mt('print ("Hello World!%N")\n'
+                '-- Comment here\n',
+                bf +
+                ' print ("Hello World!%N")\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Eiffel'])
+
+    # vhdl.
+    def test_80(self):
+        self.mt('assert false report "Hello World!"\n'
+                '-- Comment here\n',
+                bf +
+                ' assert false report "Hello World!"\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['vhdl'])
+
+    # COBOL.
+    """
+    def test_81_a(self):
+        self.mt('*> Comment here\n',

```

Comment:
The newer `*>` syntax is supported by the `COBOLFree` lexer. However, there's now two possible comment delimiters, `*` or `*>`, which CodeChat doesn't support. So, just remove this test.